app/services/mlops/hopsworks_service.py

```
@@ -0,0 +1,682 @@
+            # Get feature view
+            fv_key = f"{feature_view_name}_v{version}"
+            if fv_key not in self.feature_views:
+                feature_view = self.feature_store.get_feature_view(
+                    feature_view_name, version
+                )
+                self.feature_views[fv_key] = feature_view
+            else:
+                feature_view = self.feature_views[fv_key]
+
+            # Create training dataset
+            td_version = training_dataset_version or 1
```

Ruff (0.13.2)

Local variable `td_version` is assigned to but never used

Remove assignment to unused variable `td_version`

(F841)